Detectors/TPC/workflow/src/CATrackerSpec.cxx

```diff
@@ -74,7 +81,94 @@ DataProcessorSpec getCATrackerSpec(bool processMC, std::vector<int> const& input
       auto& tracker = processAttributes->tracker;
       parser = std::make_unique<ClusterGroupParser>();
       tracker = std::make_unique<o2::tpc::TPCCATracking>();
-      if (tracker->initialize(options.c_str()) != 0) {
+      
+      // Prepare initialization of CATracker - we parse the deprecated option string here,
+      // and create the proper configuration objects for compatibility.
+      // This should go away eventually.
+      
+      // Default Settings
+      float solenoidBz = -5.00668; // B-field
+      float refX = 1000.;          // transport tracks to this x after tracking, >500 for disabling
+      bool continuous = false;     // time frame data v.s. triggered events
+      int nThreads = 1;
+      bool useGPU = false;
+      bool dump = false;
+      char gpuType[1024] = "CUDA";
+      GPUDisplayBackend* display = nullptr;
+      
```

Comment:
@davidrohr @matthiasrichter: Could you add at once the GRP parameters parsing here?
````
#include "DataFormatsParameters/GRPObject.h"

const auto grp = o2::parameters::GRPObject::loadFrom("o2sim_grp.root");
if (grp) {
  LOG(INFO) << "Initializing run paramerers from GRP";
  solenoidBz *= grp->getL3Current()/30000.;
  continuous = grp->isDetContinuousReadOut(o2::detectors::DetID::TPC);
}
else {
  LOG(ERROR) << "Failed to initialize run parameters from GRP";
  // should we call fatal here?
}
````
I would keep at the moment hardcoded (default) grp file name (or add it as an option of the workflow, like in e.g. ``ITS/workflow/src/TrackerFlow.cxx``), eventually, this will be centrally substituted by the CCDB query or reading from the filename provided by the configurable names database.

Reply:
@shahor02 : Sure, will add it. Just one question: Where is the continuous mode set? During o2sim? I mean, this needs to be synchronized with the setting of "--TPCtriggered" in the digitizer-workflow.

Reply:
@davidrohr ``o2-sim`` just creates GRP with participating detectors and field data. Then ``o2-sim-digitizer-workflow`` updates the GRP according to readout options requested for each detector, i.e. whether the  ``grp->isDetContinuousReadOut(o2::detectors::DetID::TPC);`` returns ``false`` (``true``) depends exactly on the presence (absence) of ``--TPCtriggered`` in the digitization.